lms/static/sass/_build-lms-v2.scss

```diff
@@ -3,7 +3,22 @@
 // Version 2 - introduces the Pattern Library
 
 
-// Configuration
```

Comment:
Nit: can we go back to the simpler comments. I find Brian's section separators to be ugly. :-)